frontend/integration-tests/protractor.conf.ts

```diff
@@ -105,23 +105,23 @@ export const config: Config = {
       'tests/source-to-image.scenario.ts',
       'tests/deploy-image.scenario.ts',
       'tests/performance.scenario.ts',
+      'tests/operator-hub/operator-hub.scenario.ts',
       'tests/olm/descriptors.scenario.ts',
       'tests/olm/catalog.scenario.ts',
-      'tests/operator-hub/operator-hub.scenario.ts',
       'tests/olm/prometheus.scenario.ts',
       'tests/olm/etcd.scenario.ts'],
     all: ['tests/base.scenario.ts',
       'tests/crud.scenario.ts',
       'tests/overview/overview.scenareio.ts',
       'tests/secrets.scenario.ts',
-      'tests/olm/**/*.scenario.ts',
       'tests/service-catalog/**/*.scenario.ts',
       'tests/filter.scenario.ts',
       'tests/modal-annotations.scenario.ts',
       'tests/source-to-image.scenario.ts',
       'tests/deploy-image.scenario.ts',
       'tests/operator-hub/operator-hub.scenario.ts',
-      'tests/developer-catalog.scenario.ts'],
+      'tests/developer-catalog.scenario.ts',
```

Comment:
Should ordering matter?  Does that reveal a dependency between some of the tests?

Reply:
Wondering if we need a `FIXME` for isolation?

Reply:
I haven't made that determination yet. Just a thought that I'm testing out.